cpp/src/arrow/dataset/file_parquet_test.cc

```diff
@@ -390,6 +390,38 @@ TEST_P(TestParquetFileFormatScan, ScanRecordBatchReaderWithVirtualColumn) {
 TEST_P(TestParquetFileFormatScan, ScanRecordBatchReaderWithDuplicateColumn) {
   TestScanWithDuplicateColumn();
 }
+TEST_P(TestParquetFileFormatScan, ScanRecordBatchReaderWithFieldPathFilter) {
+  TestScanWithFieldPathFilter();
+}
+TEST_P(TestParquetFileFormatScan, ScanRecordBatchReaderWithFieldPathMultiFileFilter) {
```

Comment:
Can we make this a "common" test too?

Reply:
this is WIP test case @lidavidm I haven't finalized it yet. Still addressing the review.